swift/Lottie.swift

```diff
@@ -3,20 +3,6 @@ import CoreMedia
 
```

Comment:
The diff for this file is a bit messy. I'd recommend just looking at the file itself.